stubs/setuptools/@tests/stubtest_allowlist.txt

```diff
@@ -42,15 +42,106 @@ setuptools._distutils.dist.Distribution.get_requires
+setuptools._distutils.command.build_scripts
+setuptools._distutils.command.check
+setuptools._distutils.command.clean
+setuptools._distutils.command.install
+setuptools._distutils.command.install_data
+setuptools._distutils.command.install_headers
+setuptools._distutils.command.install_lib
+setuptools._distutils.command.install_scripts
+setuptools._distutils.command.install.HAS_USER_SITE
+setuptools._distutils.command.install.INSTALL_SCHEMES
+setuptools._distutils.command.install.SCHEME_KEYS
+setuptools._distutils.command.install.WINDOWS_SCHEME
+setuptools._distutils.command.install_lib.PYTHON_SOURCE_EXTENSION
+setuptools._distutils.command.sdist
+setuptools._distutils.command.sdist.show_formats
+setuptools._distutils.command.register
+setuptools._distutils.command.upload
+setuptools._distutils.config.DEFAULT_PYPIRC
+setuptools._distutils.dist.DistributionMetadata.set_classifiers
+setuptools._distutils.dist.DistributionMetadata.set_keywords
+setuptools._distutils.dist.DistributionMetadata.set_platforms
+setuptools._distutils.dist.fix_help_options
+setuptools._distutils.extension.read_setup_file
+setuptools._distutils.filelist.findall
+setuptools._distutils.filelist.glob_to_re
+setuptools._distutils.filelist.translate_pattern
+setuptools._distutils.sysconfig.BASE_EXEC_PREFIX
+setuptools._distutils.sysconfig.BASE_PREFIX
+setuptools._distutils.sysconfig.IS_PYPY
+setuptools._distutils.sysconfig.build_flags
+setuptools._distutils.sysconfig.expand_makefile_vars
+setuptools._distutils.sysconfig.get_python_version
+setuptools._distutils.sysconfig.parse_config_h
+setuptools._distutils.sysconfig.parse_makefile
+setuptools._distutils.sysconfig.project_base
+setuptools._distutils.sysconfig.python_build
+setuptools._distutils.util.MACOSX_VERSION_VAR
+
+# Missing submodules from setuptools._distutils
+# (Many of these may be implementation details,
+# but they can be added if people ask for them)
+setuptools._distutils.bcppcompiler
+setuptools._distutils.command.bdist
+setuptools._distutils.command.bdist_dumb
+setuptools._distutils.command.build_scripts
+setuptools._distutils.command.check
+setuptools._distutils.command.clean
+setuptools._distutils.command.config
+setuptools._distutils.command.install_data
+setuptools._distutils.command.install_egg_info
+setuptools._distutils.command.install_headers
+setuptools._distutils.command.py37compat
+setuptools._distutils.core
+setuptools._distutils.cygwinccompiler
+setuptools._distutils.debug
+setuptools._distutils.dir_util
+setuptools._distutils.fancy_getopt
+setuptools._distutils.file_util
+setuptools._distutils.log
+setuptools._distutils.msvc9compiler
+setuptools._distutils.msvccompiler
+setuptools._distutils.py38compat
+setuptools._distutils.py39compat
+setuptools._distutils.spawn
+setuptools._distutils.text_file
+setuptools._distutils.unixccompiler
+setuptools._distutils.version
+setuptools._distutils.versionpredicate
+
+# Reexported from setuptools._distutils; problems should be fixed there
+distutils\..+
```

Comment:
Good call. Should make stubtest errors clearer and contributors will be less tempted to try to apply a fix in stdlib/distutils (I know I have in the past)